modules/apps/item-selector/item-selector-criteria-api/src/main/java/com/liferay/item/selector/criteria/info/item/criterion/InfoItemItemSelectorCriterion.java

```diff
@@ -50,13 +60,38 @@ public void setMimeTypes(String[] mimeTypes) {
 		_mimeTypes = mimeTypes;
 	}
 
+	/**
+	 * @deprecated As of Athanasius (7.3.x), replaced by {@link
+	 *             #setStatuses(int...)}
+	 */
+	@Deprecated
 	public void setStatus(int status) {
 		_status = status;
+		setStatuses(status);
```

Comment:
`setStatus()` is called in a plethora of other places, so we opt to deprecate it in favor of `setStatuses()`. We also modify `setStatus()` to call `setStatuses()` in order to maintain parity between `_status` (also deprecated now) and `_statuses`. For these same reasons, this approach is also taken for `QueryDefinition`'s `_status` and `_isExcludeStatus` variables.